target/linux/pistachio/marduk/profiles/marduk_ca8210.mk

```diff
@@ -28,7 +28,8 @@ define Profile/marduk_ca8210
     PACKAGES:=kmod-i2c kmod-cascoda kmod-sound-pistachio-soc \
               wpan-tools tcpdump uhttpd uboot-envtools \
               alsa-lib alsa-utils alsa-utils-tests i2c-tools \
-              iw hostapd wpa-supplicant kmod-uccp420wlan kmod-cfg80211
+              iw hostapd wpa-supplicant kmod-uccp420wlan kmod-cfg80211 \
+              e2fsprogs
```

Comment:
if we need this for all profiles, better to move into target.mk DEFAULT_PACKAGES?